controllers/tasks.js

```diff
@@ -401,6 +401,27 @@ const assignTask = async (req, res) => {
   }
 };
 
+const updateOldTaskStatus = async (req, res) => {
+  try {
+    const allTasks = await tasks.fetchTasks();
+    const allOldTasks = allTasks.filter(
+      (task) => task.status.toUpperCase() === "COMPLETED" || task.status === "unassigned"
+    );
```

Comment:
why only capital case for COMPLETED . There can be tasks with status COMPLETED in small case.

Reply:
Just didn't want to write `task.status === 'completed' || task.status === 'COMPLETED'` so made it into `task.status.toUpperCase() === 'COMPLETED'`.

Before navigating to the route.
<img width="330" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/77037622/0e62ebeb-11a8-40eb-86b7-8ce6bb2a2f40">

After navigating to the route.
<img width="309" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/77037622/5272a7c3-ae43-4f63-b3e7-d292ea6a240e">

Reply:
can we do the same for `unassigned` ?